features-support/hooks/before.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+/* eslint new-cap: 0 */
+
+module.exports = function beforeHooks() {
+
+  // Remove any old test data.
+  this.Before(function(callback) {
```

Comment:
This will become problematic when we want to run tests in parallel in the future